docker/config.ini.tpl

```diff
@@ -11,6 +11,7 @@ eclipse.p2.profile=DefaultProfile
 eclipse.p2.data.area=@config.dir/../p2
 equinox.use.ds=true
 org.eclipse.equinox.simpleconfigurator.configUrl=file\:org.eclipse.equinox.simpleconfigurator/bundles.info
+org.opentosca.container.engine.ia.publicip={{ .Env.PUBLIC_IP }}
```

Comment:
Why do we need an additional environment variable? We already have `ENGINE_IA_HOSTNAME` which is basically intended to be the public hostname/IP address of the IA engine.

Btw, I would have called it `org.opentosca.container.engine.ia.hostname`...

Reply:
So we should just set this to a public IP? There will be no side-effects between the docker containers e.g. engine-ia:8080 whatever?